src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/header.html.twig

```diff
@@ -1,6 +1,21 @@
 <div id="header">
     <h1><a href="{{ path('_profiler_home') }}">{{ source('@WebProfiler/Icon/symfony.svg') }} Symfony Profiler</a></h1>
 
+    <div class="sf-tabs" data-processed="true">
```

Comment:
Why adding this ` data-processed="true"` ?

Reply:
Don't know, this is copy-pasted :), @javiereguiluz will handle refactoring the templates part in another PR.